libs/deps/mas/mas.js

```diff
@@ -1,4 +1,1695 @@
+        outline: none;
+        box-sizing: border-box;
+        box-shadow: inset 0 0 0 2px var(--color-accent);
+    }
+
+    .invisible {
+        visibility: hidden;
+    }
+
+    :host(:hover) .invisible {
+        visibility: visible;
+        background-image: var(--ellipsis-icon);
+        cursor: pointer;
+    }
+
+    .action-menu.always-visible {
+        visibility: visible;
+        background-image: var(--ellipsis-icon);
+    }
+
+    :host([variant='mini-compare-chart']) > slot:not([name='icons']) {
+        display: block;
+    }
+
+    .top-section {
+        display: flex;
+        justify-content: flex-start;
+        align-items: flex-start;
+        gap: 16px;
+    }
+
+    .top-section.badge {
+        min-height: 32px;
+    }
+
+    .body {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        height: 100%;
+        gap: var(--consonant-merch-spacing-xxs);
+        padding: var(--consonant-merch-spacing-xs);
+    }
+
+    footer {
+        display: flex;
+        justify-content: flex-end;
+        box-sizing: border-box;
+        align-items: flex-end;
+        width: 100%;
+        flex-flow: wrap;
+        gap: var(--consonant-merch-spacing-xs);
+
+        padding: var(--consonant-merch-spacing-xs);
+    }
+
+    hr {
+        background-color: var(--color-gray-200);
+        border: none;
+        height: 1px;
+        width: auto;
+        margin-top: 0;
+        margin-bottom: 0;
+        margin-left: var(--consonant-merch-spacing-xs);
+        margin-right: var(--consonant-merch-spacing-xs);
+    }
+
+    div[class$='-badge'] {
+        position: absolute;
+        top: 16px;
+        right: 0;
+        font-size: var(--type-heading-xxs-size);
+        font-weight: 500;
+        max-width: 180px;
+        line-height: 16px;
+        text-align: center;
+        padding: 8px 11px;
+        border-radius: 5px 0 0 5px;
+    }
+
+    div[class$='-badge']:dir(rtl) {
+        left: 0;
+        right: initial;
+        padding: 8px 11px;
+        border-radius: 0 5px 5px 0;
+    }
+
+    .body .catalog-badge {
+        display: flex;
+        height: fit-content;
+        flex-direction: column;
+        width: fit-content;
+        border-radius: 5px;
+        position: relative;
+        top: 0;
+        margin-left: var(--consonant-merch-spacing-xxs);
+    }
+
+    .detail-bg-container {
+        right: 0;
+        padding: var(--consonant-merch-spacing-xs);
+        border-radius: 5px;
+        font-size: var(--consonant-merch-card-body-font-size);
+        margin: var(--consonant-merch-spacing-xs);
+    }
+
+    .action-menu {
+        display: flex;
+        width: 32px;
+        height: 32px;
+        position: absolute;
+        top: 16px;
+        right: 16px;
+        background-color: #f6f6f6;
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: 16px 16px;
+    }
+    .hidden {
```

Comment:
is this CSS global?

Reply:
@narcis-radu no applies in the shadow root of the merch-card web component.
see: https://github.com/adobecom/milo/blob/stage/libs/features/mas/web-components/src/merch-card.css.js#L171

That's to good thing about web components, you can write simple css rules without worrying about conflicts with other elements.